### PR TITLE
UHM-4354 (Pt 1, just Mexico) Move Location, Location Tag, and Lcation Attribute Types installation out of PIH core and into config repos

### PR DESCRIPTION
**Pull request by brandones**
A first step toward https://pihemr.atlassian.net/browse/UHM-4354 . This moves Location Tags and Attribute Types out to configuration. It completely moves Mexico locations out to configuration.

I tested local servers with both CES and ZL HIV configurations.

Depends on https://github.com/PIH/openmrs-config-pihemr/pull/162 and https://github.com/PIH/openmrs-config-ces/pull/24 and https://github.com/PIH/openmrs-module-mirebalais/pull/147

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import org.springframework.stereotype.Component;
 
 @Component
-@Requires({ LocationTagBundle.class, LocationAttributeTypeBundle.class} )
+@Requires({ LocationAttributeTypeBundle.class} )
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java:9]: I saw you added the attribute types to Iniz in config-pihemr... is there a reason you are still installing them here?

**Review by mogoodrich — COMMENTED** [review]
One question, but generally looks good.

I like the approach of moving the Location Tags and Attributes out and starting with a single country (Mexico), confirming that works, and then moving on to other countries from there.

We also should test a Mexico install from "scratch" (though that could happen after merging)

File: api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java (hunk 1 above)
Comment (brandones) [reply api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java:9]: Oversight. Thanks!

**Review by mseaton — APPROVED** [review]
Nice!  LGTM.

**Pull request merged**